tests/test_core.py

```diff
@@ -1030,19 +1030,15 @@ def test_exceptions_minimal_runtime(self):
     self.set_setting('EXIT_RUNTIME')
     self.maybe_closure()
     self.set_setting('MINIMAL_RUNTIME')
+    self.emcc_args += ['--pre-js', test_file('minimal_runtime_support.js')]
```

Comment:
why is this needed here but not in other minimal runtime tests?

Reply:
I think it would be good to add both `EXIT_RUNTIME` and this support file to all `MINIMAL_RUNTIME` tests, but its not strictly needed today.

If you look at `MINIMAL_RUNTIME` today, without this change, calls to `exit()` (in default builds) from inside of main are silently logged and node returns 0 (even for non-zero exit() calls).   Also, return codes from main are silently ignored. 

With this change calls to exit() are no longer ignored (due to the then.catch() change) and returns from main are still ignored (unless one sets `EXIT_RUNTIME`).

So basically any test that needs to accurately detect the exit code should set `EXIT_RUNTIME` and use `minimal_runtime_support.js`... I'm guessing these are the only tests that use `MINIMAL_RUNTIME` and expect a non zero return code.

Reply:
I see. Sounds reasonable. How about renaming `minimal_runtime_support.js`, though, to a name that indicates this is for support of EXIT_RUNTIME specifically?

Reply:
Done